### PR TITLE
CVE-2010-2227

### PR DESCRIPTION
**Pull request by spc7641**
CVE-2010-2227 information, for the vcc the commits were old enough that I could not find the commit ids.

### CODE REVIEW

File: cves/CVE-2010-2227.yml [hunk 1]
```diff
@@ -141,8 +149,11 @@ interesting_commits:
     section by explaining what happened between the VCCs and the fix.
   answer:
   commits:
-    - commit:
-      note:
+    - commit: 2ee172f418591edd61cfd3dc9676753b281d1fe2
+      note: |
+        SVN rev 802162. This commit shows that while moving up in the versions this vunerability still existed and would not be found for
+        around a year from this commit. This section of code had not been touched since the vunerability was created, so this was the first
+        time since its creation that a developer could have seen the innapro
```
Comment (Scar56) [inline cves/CVE-2010-2227.yml]: sentence cut off?

**Comment by Scar56** [comment]
git commit hashes can be found with git log --grep="<svn commit>", the git hash will be shown at the top
upvotes 3

File: cves/CVE-2010-2227.yml [hunk 2]
```diff
@@ -43,15 +43,18 @@ description_instructions: |
...
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
 bounty:
   amt:
-  announced: 2010-07-13
+  announced:
   url:
 bugs: []
 fixes_vcc_instructions: |
```
Comment (ADAdebomi) [inline cves/CVE-2010-2227.yml:60]: Possibly add more details as to what the fix changed

File: cves/CVE-2010-2227.yml [hunk 3]
```diff
@@ -65,9 +68,11 @@ fixes:
      note:
 vccs:
   - commit:
-    note:
+    note: SVN rev 299033, from the tomcat website. Commit id was not found in the git logs
```
Comment (ADAdebomi) [inline cves/CVE-2010-2227.yml]: What was changed to make this a VCC? The commits can be found using git log --pretty="%H" --grep="trunk@892795" where 892795 is replaced whatever revision number the commit is

File: cves/CVE-2010-2227.yml [hunk 4]
```diff
@@ -10,21 +10,21 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
```
Comment (ADAdebomi) [inline cves/CVE-2010-2227.yml:10]: Upvote : 4

File: cves/CVE-2010-2227.yml (hunk 3 above)
Comment (spc7641) [reply cves/CVE-2010-2227.yml]: For the commits, all of the vccs that I am using are from 2003 or 2002, and the farthest back in the git logs of tomcat that I could find was from 2006.

**Pull request merged**